lib/matplotlib/axes/_axes.py

```diff
@@ -2942,6 +2942,17 @@ def pie(self, x, explode=None, labels=None, colors=None,
         shadow : bool, default: False
             Draw a shadow beneath the pie.
 
+        normalize: None or bool, default: None
+            ``pie()`` used to draw a partial pie if ``sum(x) < 1``. This
```

Comment:
Can you change this state what `normalize` does _first_, and then the deprecated information in a second paragraph?